### DISCUSSION TITLE
Docker implementation?

### DISCUSSION THREAD

**Discussion by lhassell** [every-day-things/citadel#45] [Ideas]
Has anyone tried spinning up a docker version?

**Comment by phildenhoff** [comment]
What would you expect from a docker version? It’s a desktop app 😁

There’s a branch to turn it into a web app that uses the browser file system api, but it’s not really a priority for me. I prefer the standalone desktop app

**Comment by lhassell** [comment]
To be honest, I was thinking this was a replacement for Calibre-Web. Looking for ebook management that supports different libraries and could allow for people to have varying levels of access. Use case was for my library of RPG books for my play group.

**Reply by phildenhoff** [reply]
Where does Calibre-Web fall short for managing your RPG book library for your group?

I'm mostly focused on refining Citadel as a desktop app, like I said, but it is certainly possible to support running it as a web app. However, shared/non-personal library management requires a lot more consideration — user support, managing access (like you said), security/privacy concerns, UI changes to support multiple sub-libraries... Citadel isn't a tool for managing organization-level access to ebooks. But I'm curious if this is a gap you see — do you know if there's a tool that satisfies this niche? Have you heard about the same problem from others?
